### PR TITLE
Implemented Filters for proposals and authorities tables

### LINKED ISSUE #630: [TASK] Filter functionality of the proposal table on the Proposal Page
The proposal table on the proposal page does not have the filter functionality at the moment. We should implement the UI along with the filter functionality to work with subql backend.

### Tasks

- [x] Implement the Filter UI on Proposal Table
- [x] Integrate the filter functionality with the subql backend.

### Relevant resources

- [Figma design](https://www.figma.com/file/UrIhW5ZikdiIyWWi5XKp9m/%F0%9F%8C%90--Statistics-Page-%5Bhand-off%5D?node-id=2%3A62&viewport=3558%2C-1288%2C0.36)

### LINKED ISSUE #631: [TASK] Filter functionality of the authority table on the Authority Page
The authority table on the authority page does not have the filter functionality at the moment. We should implement the UI along with the filter functionality to work with the subql backend.

### Tasks

- [x] Implement the Filter UI on Authority Table
- [x] Integrate the filter functionality with the subql backend.

### Relevant resources

- [Figma design](https://www.figma.com/file/UrIhW5ZikdiIyWWi5XKp9m/%F0%9F%8C%90--Statistics-Page-%5Bhand-off%5D?node-id=2%3A7026&viewport=1855%2C540%2C0.19)

### PR DESCRIPTION
**Pull request by AhmedKorim**
## Summary of changes
- Implemented Auth page filters
	- Fetch the locations from the SubQuery node
	- Filter the authorities by the location 
	- Filter the authorities by the uptime and reputation
	- Add the function for filtering the authorities by account id, authority id
- Implemented Proposals page filters
	- Filter the proposals by type and status
	- Filter the proposals by the tx hashes 
- Fixes on authorities fetching fetch the session validators instead of validators
- Display UI values for reputation and uptime from the API
- Fix for the chains config , the icon field had the type `any`
- Fix SVG icons fixed sizes
- Add a new component for toggle checkbox menus 

### Reference issue to close (if applicable)
- Closes #630 
- Closes #631 
-----
### Code Checklist 

https://user-images.githubusercontent.com/35463181/195913835-b0884c90-fbe2-4458-a4e6-9df07711db57.mp4

- [ ] Tested
- [ ] Documented

### CODE REVIEW

File: packages/page-statistics/src/containers/AuthoritiesTable/AuthoritiesTable.tsx [hunk 1]
```diff
@@ -105,19 +119,85 @@ export const AuthoritiesTable: FC<AuthoritiesTableProps> = ({ data: dataProp })
...
+  const headers = useMemo(
+    () => table.getHeaderGroups().map((headerGroup) => headerGroup.headers.map((header) => header)),
+    [table]
+  );
+
+  const [{ column: keygenFilterCol }, { column: signatureFilterCol }] = useMemo(
+    () => headers[0].filter((header) => header.column.getCanFilter()),
+    [headers]
+  );
+  console.log({ authorities });
```
Comment (dutterbutter) [inline packages/page-statistics/src/containers/AuthoritiesTable/AuthoritiesTable.tsx:141]: Please remove the console log here.

**Review by dutterbutter — CHANGES_REQUESTED** [review]
Can you please remove the `console.log` that is polluting the console:
```
Active session 134
134
```

Also it seems something has changed in this pr as I am receiving a 404 from backend yet the backend seems to working fine.

**Comment by AhmedKorim** [comment]
@dutterbutter  there is a PR on SubQuery side that need to be deployed

**Comment by dutterbutter** [comment]
> @dutterbutter there is a PR on SubQuery side that need to be deployed

done. @AhmedKorim

**Comment by dutterbutter** [comment]
@AhmedKorim deployed updated backend resolved the 400 resource issue. If you could please remove the following console logs:
Screen Shot 2022-10-17 at 3 34 15 PM
Screen Shot 2022-10-17 at 3 34 18 PM

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**